.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,19 @@
+name: Benchmarks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  Benchmarks:
+    runs-on: ubuntu-latest
+    container: ghcr.io/notional-labs/cosmos
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.19' # The Go version to download (if necessary) and use.
+      - name: run benchmarks
+        run: make bench
```

Comment:
do we really want to run benchmarks on CI? What's the goal? It takes lot of cycles. If we don't have performance DIFF reporting, then running these benchmark is burining CI cycles.